src/views/ContinuationOut.vue

```diff
@@ -62,7 +62,7 @@
             </header>
 
             <!-- Ledger Detail -->
-            <section>
+            <section v-if="isRoleStaff">
```

Comment:
Per the clarified requirements from Naveen, this component (and all its associated variables) should be removed.

Also, this is a staff-only filing, so this code did nothing.